nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/TailFile.java

```diff
@@ -131,7 +131,7 @@
             .displayName("Base directory")
             .description("Base directory used to look for files to tail. This property is required when using Multifile mode.")
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
-            .addValidator(StandardValidators.FILE_EXISTS_VALIDATOR)
+            .addValidator(StandardValidators.DIRECTORY_EXISTS_VALIDATOR)
```

Comment:
@tlsmith109 A new ticket should be opened if we want to modify TailFile as well as the ticket is specific to FetchSFTP

That being said, it looks like there is code that already makes sure this is a directory [1].  Although, at first glance I am unsure why this wasn't handled here, as your code is suggesting. 

[1] https://github.com/apache/nifi/blob/main/nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/TailFile.java#L277

Reply:
Didn't realize I left this in here, will remove